packages-native/safe-area-view/src/SafeAreaView.tsx

```diff
@@ -0,0 +1,20 @@
+import { createElement, Fragment } from "react";
+import { SafeAreaView as ReactSaveAreaView, View } from "react-native";
+import { flattenStyles } from "@native-mobile-resources/util-widgets";
+
+import { SafeAreaViewStyle, defaultSafeAreaViewStyle } from "./ui/Styles";
+import { SafeAreaViewProps } from "../typings/SafeAreaViewProps";
+
+export const SafeAreaView = (props: SafeAreaViewProps<SafeAreaViewStyle>): JSX.Element => {
+    const customStyles = props.style.filter(o => o != null);
+    const styles = flattenStyles(defaultSafeAreaViewStyle, customStyles);
+
+    return (
+        <Fragment>
+            <ReactSaveAreaView style={{ flex: 0, ...styles.unsafeAreaTop }} />
+            <ReactSaveAreaView style={{ flex: 1, ...styles.unsafeAreaBottom }}>
```

Comment:
Why are we adding the content to the bottom unsafe area? What happen if you have some content that should be full screen??